JAGPropertyConverter/JAGPropertyConverter.m

```diff
@@ -285,6 +285,17 @@ - (NSDictionary*) convertToDictionary: (id) model {
         // set value in dictionary
         [values setValue:[self decomposeObject: object] forKey:propertyName];
     }
+
+    // Add all custom keypaths defined for the model.
+    if (customMapping) {
+        for (NSString *customKey in customMapping) {
+            BOOL isKeyPath = [customKey rangeOfString:@"."].location != NSNotFound;
```

Comment:
extract this into own private method: `- (BOOL)_isKeyPathKey:(NSString *)key;` or similar